### PR TITLE
[Chore] Update Readme

### PR DESCRIPTION
**Pull request by Wadeewee**
## What happened 👀

Add the section on how to use the app in README

## Insight 📝

- Currently, The `API` doesn't return `display_price` and `price` on this endpoint `/products.json`. Therefore, we need to mock them in the `Model` class to make the app render properly. For more info please check this [thread](https://nimble-co.slack.com/archives/C05ES4C8AN7/p1689915325083709)
- Calculate the total price from `display_price` instead of `price`.

## Proof Of Work 📹

screencapture-github-nimblehq-showcase-flutter-build-web-desktop-blob-f0d69cf0f021cabd2032e74b15799dae042ea364-README-md-2023-07-27-12_02_47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
-Building Web and Desktop (Windows) apps with Flutter
+Building Web and Desktop (Windows) apps with Flutter:
+
+- Welcome screen:
+
+  <img width="1049" alt="Screen Shot 2566-07-27 at 09 39 20" src="https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/0900f56b-889f-4a0b-90f9-2044d377e819">
+
+- Main screen:
+
+  <img width="1049" alt="Screen Shot 2566-07-27 at 11 12 50" src="https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/e76393f8-65f1-4d76-b5bb-7db46941fa3a">
```
Comment (minhnimble) [inline README.md:11]: Do we really need to set the width for the images here? Can't we leave them auto-scaled? 🤔

File: README.md [hunk 2]
```diff
@@ -58,6 +69,17 @@ To run the app on the web, follow these steps:
 >It provides additional information and troubleshooting steps specifically for resolving these issues
 >encountered during local development.
 
+### How to use the app:
+
+1. Open the app and click on the `Sign in with CX` button.
+2. On the main screen, select products to add to your cart.
+3. On the right side of the main screen in the cart section:
+   - You can click on bin button to remove an item from the cart list.
```
Comment (minhnimble) [inline README.md:77]: Wording highlight improvement:
```suggestion
   - You can click the `Bin` button to remove an item from the cart list.
```

File: README.md
Comment (kaungkhantsoe) [inline README.md:1]: > Create these `.env` files in the root directory and add the required
  environment variables into them.

Shall we add an example of the required fields for `.env` file?

File: README.md (hunk 2 above)
Comment (minhnimble) [inline README.md:77]: Can we shorten these sentences into 1 step like others? For example:
```suggestion
3. To remove a product from the cart list in the cart, select the `Bin` button on the right side of that product.
```

**Review by kaungkhantsoe — COMMENTED** [review]
The rest LGTM

File: README.md
Comment (Wadeewee) [reply README.md:1]: What about something like this?

```env
REST_API_ENDPOINT=
NIMBLE_CX_USERNAME=
NIMBLE_CX_PASSWORD=
ACCESS_TOKEN=
```

File: README.md (hunk 1 above)
Comment (Wadeewee) [reply README.md:11]: Let me check @minhnimble 🏃‍♀️

File: README.md
Comment (kaungkhantsoe) [reply README.md:1]: @Wadeewee Please correct me if I am wrong, I have the following environment variables 👇 . Did you intentionally remove the rest? 🙏  The idea is for the developers to know which environment variables are needed later.  
```
CLIENT_ID=
CLIENT_SECRET=
REST_API_ENDPOINT=
USERNAME=
PASSWORD=
NIMBLE_CX_USERNAME=
NIMBLE_CX_PASSWORD=
ACCESS_TOKEN=
```

**Review by kaungkhantsoe — APPROVED** [review]
LGTM

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**